doc/admin-guide/plugins/escalate.en.rst

```diff
@@ -61,4 +61,4 @@ With this line in :file:`remap.config` ::
 Traffic Server would accept a request for ``cdn.example.com`` and, on a cache miss, proxy the
 request to ``origin.example.com``. If the response code from that server is a 401, 404, 410,
 or 502, then Traffic Server would proxy the request to ``second-origin.example.com``, using a
-Host: header of ``cdn.example.com``.
\ No newline at end of file
+Host: header of ``cdn.example.com``.
```

Comment:
Can you please `git checkout -- doc/admin-guide/plugins/esclate.en.rst` as I don't think you meant to change this file.